src/Admin/TerritorialCouncil/TerritorialCouncilMembershipAdmin.php

```diff
@@ -67,6 +68,37 @@ protected function configureDatagridFilters(DatagridMapper $filter)
                     return true;
                 },
             ])
+            ->add('pcQualities', CallbackFilter::class, [
+                'show_filter' => true,
+                'label' => 'Qualité au CoPol',
+                'field_type' => ChoiceType::class,
+                'field_options' => [
+                    'choices' => TerritorialCouncilQualityEnum::ALL_POLITICAL_COMMITTEE_QUALITIES,
+                    'choice_label' => function (string $choice) {
+                        return "political_committee.membership.quality.$choice";
+                    },
+                    'multiple' => true,
+                ],
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
+                    if (!$value['value']) {
+                        return false;
+                    }
+
+                    if (!\in_array('adherent', $qb->getAllAliases())) {
+                        $qb
+                            ->leftJoin("$alias.adherent", 'adherent')
+                            ->leftJoin('adherent.politicalCommitteeMembership', 'pcMembership')
+                        ;
+                    }
+                    $qb
+                        ->leftJoin('pcMembership.qualities', 'pcQuality')
```

Comment:
this should be into the if above non ? if we don't enter the if, pcMembership seems not defined